recipes/openassetio/all/test_v1_package/conanfile.py

```diff
@@ -0,0 +1,83 @@
+import pathlib
```

Comment:
Please, try to simplify it to a minimal executable. The idea is testing the generated package, not an unit test or a full example.

Reply:
The test project does only check linkage, `#include`s, etc and is not a full example/test.  It's used in upstream CI too so seemed worth adhering to DRY and re-using it.  

Is the main concern that the test project is pulled externally rather than bundled?  I did wonder how ConanCenter would view that. We can bundle it alongside the recipe if that makes things easier.

Reply:
Not entire true, you need to download an external project to execute your test. Here, we just need a small C++ file with few lines of code proving a header include and library linkage. Downloading an external example is too much. Please, reduce it to a single cpp file, you don't need to add those extras requirements like cmake and cpython, you could call a method which prints the library version for instance, or a class constructor.

Reply:
Removed the external fetch of test project and bundled a simplified version instead in abf1234d